### ISSUE TITLE
Ignoring punctuation marks in text search

### ISSUE THREAD

**Issue by nadavtsa** [RediSearch/RediSearch#3204]
Hi, 
is there a way to disable the breaking of text queries into terms by punctuation marks?

We have a lot of fields containing punctuation marks (mostly UUID's which contain dashes "-")

**Comment by gkorland** [comment]
Please see [Tokenization](https://redis.io/docs/stack/search/reference/escaping/)

**Comment by nadavtsa** [comment]
@gkorland Yes, I saw that, I was wondering if there is a more elegant way to do that (without adding slashes).

Anyway, can you explain why punctuation marks are not tokenized as well?
